_layouts/default.html

```diff
@@ -1,80 +1,84 @@
 <!DOCTYPE html>
```

Comment:
Somehow this file got autoformatted. Let me know if this is a problem!

Reply:
I don't see this as an issue.